doc/manual/manual.tex

```diff
@@ -7795,6 +7795,43 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\subsubsection{The Bunge et al. mantle convection experiments}
+
+\textit{This section was contributed by Cedric Thieulot and Bob Myhill.}
+
+Early mantle modelling studies of the 70's, 80's and 90's were often concerned with simple setups (Cartesian geometries, incompressible fluid, free slip boundary conditions) and investigated the influence of the Rayleigh number, the heating mode or the temperature dependence of the viscosity on temperature, pressure and/or strain rate \cite{youn74,buss75,buss79,BBC89,BC93,vavy93,burb97}. In this cookbook, we use the `simple' material model to reproduce the setup in \cite{burb96}, which reported that even modest increases in mantle viscosity with depth could have a marked effect on the style of mantle convection.
+
+Although the original article showcases results obtained in a 3D hollow sphere, we here run the models in an annular domain of inner radius R$_i$=3480 \si{\kilo\meter} and outer radius  R$_o$ = 6370 \si{\kilo\meter}.
+The surface temperature is set to $T_{\textit{surf}}$ = 1060 \si{\kelvin} and 
+the bottom temperature to $T_{\textit{cmb}}$ = 3450 \si{\kelvin}
+The gravity vector is radial and its magnitude is $g$ = 10 \si{\metre\per\second\squared}.
+
+There is a single incompressible fluid in the domain, characterised by $\rho_0$ = 4500 \si{\kilogram\per\metre\cubed}, $\alpha$ = $2.5\cdot10^{-5}$ \si{\per\kelvin}, 
+$k$ = 4 \si{\watt\per\metre\per\kelvin},
```

Comment:
same here and everywhere below -- put the whole formula, including units, into the math environment. This makes sure the spacing around all math symbols is correct